src/psyclone/tests/psyir/backend/psyir_openmp_test.py

```diff
@@ -245,3 +246,39 @@ def test_gocean_omp_do():
   a = b;
 }'''
     assert correct in result
+
+
+# ----------------------------------------------------------------------------
+def test_gocean_omp_taskloop():
+    '''Test that an OMP Taskloop directive in a 'classical' API (gocean here)
+    is created correctly.'''
+    _, invoke = get_invoke("single_invoke.f90", "gocean1.0",
+                           idx=0, dist_mem=False)
+    omp = OMPTaskloopTrans()
+    _, _ = omp.apply(invoke.schedule[0])
```

Comment:
I think you also need to manually add a TaskWait node to this schedule?

Reply:
Yep sorry, brain tired and wrote the test for the wrong Task directive. I fixed this now, the test covers both of them as it was hard for me to work out a standalone test for just TaskWait

Reply:
Thanks.